configuration/salt.md

```diff
@@ -321,91 +343,88 @@ domains this way.
+### `qvm.running`
 
-Ensures the domain is running:
+Ensures the specified domain is running:
 
     domain is running:
       qvm.running:
         - name: salt-test4
 
-## qubes pillar module
+## The `qubes` Pillar Module
 
-Additional pillar data is available to ease targeting configuration (for
-example all the templates). List here may be subject to changes in future
-releases.
+Additional pillar data is available to ease targeting configurations (for
+example all templates). 
+***Note*** List here may be subject to changes in future releases.
 
-### qubes:type
+### `qubes:type`
 
 VM type. Possible values:
 
- - `admin` - administration domain (`dom0`)
+ - `admin` - Administration domain (`dom0`)
  - `template` - Template VM
  - `standalone` - Standalone VM
- - `app` - template based AppVM
+ - `app` - Template based AppVM
 
-### qubes:template
+### `qubes:template`
 
-Template name on which given VM is based (if any).
+Template name on which a given VM is based (if any).
 
-### qubes:netvm
+### `qubes:netvm`
 
 VM which provides network to the given VM
 
 ## Debugging
-The output for each vm is logged in `/var/log/qubes/mgmt-VM_NAME.log`.
 
-If the log does not contain useful information, you can stop `qubesctl` by 
-pressing `ctrl+z`.
+The output for each VM is logged in `/var/log/qubes/mgmt-VM_NAME.log`.
 
-You need to:
-1. run `sudo qubesctl --skip-dom0 --target=VM_NAME state.highstate`
-2. When your vm is being started (yellow) press Ctrl-Z on qubesctl.
-3. Open terminal in disp-mgmt-VM_NAME.
-4. Look at /etc/qubes-rpc/qubes.SaltLinuxVM - this is what is 
-   executed in the management vm.
-5. Get the last two lines:
+If the log does not contain useful information:
+1. Stop `qubesctl` using Ctrl-z
+2. Run `sudo qubesctl --skip-dom0 --target=VM_NAME state.highstate`
+3. When your VM is being started (yellow) press Ctrl-z on qubesctl.
+4. Open terminal in disp-mgmt-VM_NAME.
+5. Look at /etc/qubes-rpc/qubes.SaltLinuxVM - this is what is 
+   executed in the management VM.
+6. Get the last two lines:
 
-    export PATH="/usr/lib/qubes-vm-connector/ssh-wrapper:$PATH"
-    salt-ssh "$target_vm" $salt_command
+    $ export PATH="/usr/lib/qubes-vm-connector/ssh-wrapper:$PATH"
+    $ salt-ssh "$target_vm" $salt_command
 
   Adjust $target_vm (VM_NAME) and $salt_command (state.highstate).
-6. Execute them, fix problems, repeat.
+7. Execute them, fix problems, repeat.
 
-## Known pitfalls
+## Known Pitfalls
 
 ### Using fedora-24-minimal
-The fedora-24-minimal package is missing the sudo package.
+
+The fedora-24-minimal package is missing the `sudo` package.
 You can install it via:
 
-    qvm-run -p vmname 'dnf install -y sudo'
+    $ qvm-run -p vmname 'dnf install -y sudo'
     
-The `-p` is will cause the execution to wait until the package is installed.
-This is important when using a state with `cmd.run`.
+The `-p` will cause the execution to wait until the package is installed.
+Having `sudo` is important when using a state with `cmd.run`.
```

Comment:
Ok, so I will replace the "having sudo" to "having the -p flag"? 
I will also add the "-u root" option.

As for the fedora-24-minimal instead of the vmname, I get what you are saying but should it be <fedora-24-minimal-template> instead?  For instance, I have a bunch of templates based off of the fedora-24-minimal base template.  Or are you thinking this would be done at the beginning before they clone off a bunch of templates?